src/ruby/yast/wfm.rb

```diff
@@ -213,6 +213,61 @@ def self.call_builtin_wrapper(*args)
             "Caller:  #{e.backtrace.first}"
     end
 
+    private_class_method def self.handle_signal_exception(e)
+      signame = Signal.signame(e.signo)
+      msg = "YaST receive signal %s and will exit.\n If killing of YaST is " \
+        "not requested then please report a bug report with logs obtained from " \
+        "save_y2logs script." % signame
```

Comment:
will check it in small terminal. Type I fix....Well part with killing probably affect only SIGTERM, but e.g. even systemd can send such signal.

And one more idea we discussed early, that maybe we should use timed popup, to kill it in reasonable time after getting sigterm